.github/workflows/test.yml

```diff
@@ -77,3 +77,22 @@ jobs:
       - run: CI=false yarn build
         env:
           CI: true
+
+  run-dotrun:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Install dotrun
+        uses: canonical/install-dotrun@main
+
+      - name: Install dependencies
+        run: /snap/bin/dotrun install
+
+      - name: Build assets
+        run: /snap/bin/dotrun build
+
+      - name: Run dotrun
+        run: |
+          /snap/bin/dotrun &
+          curl --head --fail --retry-delay 1 --retry 30 --retry-connrefused http://localhost:8400
```

Comment:
hmm, i wonder if you can do something like `wait-on` using dotrun? if we expose it in package.json to do something like `wait-on htt-get://0.0.0.0:8400` ?

How will we stop the server we started on line 97?

Reply:
I'm doing something similar in this PR: https://github.com/canonical-web-and-design/maas-ui/pull/3635/files#diff-d2632f2b1ff3d2b2f5fe8ab88039d75c14cffaa906b7b82d64ba5a89d6e90bcaR31-R37

```
      - name: Wait for MAAS
        uses: nev7n/wait_for_response@v1
        with:
          url: "${{env.MAAS_URL}}/MAAS/r"
          responseCode: 200
          timeout: 200000
          interval: 500
```

Reply:
> How will we stop the server we started on line 97?

@sparkiegeek the container will get torn down at the end of the test run so no need to specifically stop the server I don't think. Is that what you're asking about?